Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateAVFoundationObjC.mm

```diff
@@ -2726,6 +2726,12 @@ void determineChangedTracksFromNewTracksAndOldItems(MediaSelectionGroupAVFObjC*
             .height = static_cast<unsigned>(CVPixelBufferGetHeight(m_lastPixelBuffer.get())),
             .mediaTime = entry.displayTime.toDouble(),
             .presentedFrames = static_cast<unsigned>(++m_sampleCount),
+
+            // FIXME: These explicit initialization of std::optional shouldn't be necessary but clang-17 generates warnings without them.
+            .processingDuration = std::nullopt,
```

Comment:
Is it about std::optional? Or is it just because this is using designated initializer syntax and the new clang matches other compilers and requires all of those to be set?

Reply:
I'm not sure. I can't tell because all uninitialized member variables were `std::optional`.

Reply:
I would remove the comment. I don't think it adds much, especially if you aren't sure it.